packages/bot-web-ui/src/utils/__tests__/session-storage.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { getStoredItemsByKey, getStoredItemsByUser, setStoredItemsByKey } from 'Utils/session-storage';
+
+describe('Session Storage Util', () => {
+    const storageKey = 'example_key';
+    const defaultValue = 'default_value';
+    const storedItems = { example: 'data' };
+
+    it('should return default value when storage is empty', () => {
+        const result = getStoredItemsByKey(storageKey, defaultValue);
+        expect(result).toBe(defaultValue);
+    });
+
+    it('should return default value when loginid is falsy', () => {
+        const loginid = '123';
+        const result = getStoredItemsByUser(storageKey, loginid, defaultValue);
+        expect(result).toBe(defaultValue);
+    });
+
+    it('should set stored items', () => {
+        setStoredItemsByKey(storageKey, storedItems);
+        const result = getStoredItemsByKey(storageKey, defaultValue);
+        expect(result).toStrictEqual(storedItems);
+    });
+
+    it('should throw error if invalid object is passed to store', () => {
+        const consoleWarnMock = jest.fn();
+        global.console.warn = consoleWarnMock;
```

Comment:
what is global.console.warn? @shafin-deriv

Reply:
@sandeep-deriv Bhai In the function there is a `try catch` block and inside the catch block only a `console.warn` is used. As there is nothing else done I am expecting the `console.warn` to have been called.